[unitree/cross] make userを久しぶりに実行すると/bin/bashが無いと怒られて失敗する

cc:@iory
make userを久しぶりに実行すると/bin/bashがないと出てエラーになります．
解決法としては
```
docker rmi multiarch/qemu-user-static
./prepare_requirements_ros1.sh # or docker pull multiarch/qemu-user-static
```
として，multiarch/qemu-user-staticを入れ直すと直ります．

実行時の挙動は以下のとおりです．
```bash
$ make user
./build_user.sh
WARNING: Source directory is found arm64v8_User
WARNING: It might contain old data. We recommend to remove arm64v8_User and start again
WARNING: Are you sure to continue [y/N] ? y
+ mkdir -p arm64v8_User/src/jsk_robot
+ rsync -avzh --delete --exclude 'jsk_unitree_robot/cross*' --exclude go1.l --exclude go1-simple.l ../../../jsk_robot arm64v8_User/src/
sending incremental file list
deleting jsk_robot/.git/CATKIN_IGNORE
deleting jsk_robot/.github/CATKIN_IGNORE
deleting jsk_robot/.travis/CATKIN_IGNORE
deleting jsk_robot/doc/CATKIN_IGNORE
deleting jsk_robot/jsk_aero_robot/CATKIN_IGNORE
deleting jsk_robot/jsk_baxter_robot/CATKIN_IGNORE
deleting jsk_robot/jsk_denso_robot/CATKIN_IGNORE
deleting jsk_robot/jsk_fetch_robot/CATKIN_IGNORE
jsk_robot/.git/
jsk_robot/.git/COMMIT_EDITMSG
jsk_robot/.git/index
jsk_robot/.git/logs/HEAD
jsk_robot/.git/logs/refs/heads/unitree-add-accel
jsk_robot/.git/logs/refs/remotes/tkmtnt7000/unitree-add-accel
jsk_robot/.git/modules/.travis/
jsk_robot/.git/objects/
jsk_robot/.git/objects/a6/
jsk_robot/.git/objects/a6/fafb40a73d31eb34c34e15129944c99dac4f2b
jsk_robot/.git/objects/aa/
jsk_robot/.git/objects/aa/02dbeb4cb2f89edd25344cffb2fff91b344185
jsk_robot/.git/objects/ac/
jsk_robot/.git/objects/ac/acd77435756d268356df4b017d2418c5720c28
jsk_robot/.git/objects/b5/
jsk_robot/.git/objects/b5/7954f59bcce6c582b4ea94d686274ec70b9072
jsk_robot/.git/objects/ba/
jsk_robot/.git/objects/ba/05882d40b11a7ace663e66d3defe84686b9e67
jsk_robot/.git/objects/d3/
jsk_robot/.git/objects/d3/d0ae8b5b84c7a4d4dc2b971b5975b1778e78ad
jsk_robot/.git/refs/heads/
jsk_robot/.git/refs/heads/unitree-add-accel
jsk_robot/.git/refs/remotes/tkmtnt7000/
jsk_robot/.git/refs/remotes/tkmtnt7000/unitree-add-accel
jsk_robot/.github/
jsk_robot/.travis/
jsk_robot/doc/
jsk_robot/jsk_aero_robot/
jsk_robot/jsk_baxter_robot/
jsk_robot/jsk_denso_robot/
jsk_robot/jsk_fetch_robot/
deleting jsk_robot/jsk_kinova_robot/CATKIN_IGNORE
deleting jsk_robot/jsk_magni_robot/CATKIN_IGNORE
deleting jsk_robot/jsk_naoqi_robot/CATKIN_IGNORE
deleting jsk_robot/jsk_pr2_robot/CATKIN_IGNORE
deleting jsk_robot/jsk_robot/CATKIN_IGNORE
jsk_robot/jsk_kinova_robot/
jsk_robot/jsk_magni_robot/
jsk_robot/jsk_naoqi_robot/
jsk_robot/jsk_pr2_robot/
jsk_robot/jsk_robot/
jsk_robot/jsk_robot_common/jsk_robot_startup/scripts/
jsk_robot/jsk_robot_common/jsk_robot_startup/scripts/head_teleop_joy_completion.py
jsk_robot/jsk_unitree_robot/unitreeeus/

sent 129.51K bytes  received 1.29K bytes  261.60K bytes/sec
total size is 441.19M  speedup is 3,373.07
++ find arm64v8_User/src/jsk_robot -maxdepth 1 -mindepth 1 -type d
+ for dir in $(find ${SOURCE_ROOT}/src/jsk_robot -maxdepth 1 -mindepth 1 -type d)
+ [[ ! arm64v8_User/src/jsk_robot/jsk_magni_robot =~ jsk_unitree_robot|jsk_robot_common ]]
+ touch arm64v8_User/src/jsk_robot/jsk_magni_robot/CATKIN_IGNORE
+ for dir in $(find ${SOURCE_ROOT}/src/jsk_robot -maxdepth 1 -mindepth 1 -type d)
+ [[ ! arm64v8_User/src/jsk_robot/doc =~ jsk_unitree_robot|jsk_robot_common ]]
+ touch arm64v8_User/src/jsk_robot/doc/CATKIN_IGNORE
+ for dir in $(find ${SOURCE_ROOT}/src/jsk_robot -maxdepth 1 -mindepth 1 -type d)
+ [[ ! arm64v8_User/src/jsk_robot/.travis =~ jsk_unitree_robot|jsk_robot_common ]]
+ touch arm64v8_User/src/jsk_robot/.travis/CATKIN_IGNORE
+ for dir in $(find ${SOURCE_ROOT}/src/jsk_robot -maxdepth 1 -mindepth 1 -type d)
+ [[ ! arm64v8_User/src/jsk_robot/jsk_naoqi_robot =~ jsk_unitree_robot|jsk_robot_common ]]
+ touch arm64v8_User/src/jsk_robot/jsk_naoqi_robot/CATKIN_IGNORE
+ for dir in $(find ${SOURCE_ROOT}/src/jsk_robot -maxdepth 1 -mindepth 1 -type d)
+ [[ ! arm64v8_User/src/jsk_robot/.git =~ jsk_unitree_robot|jsk_robot_common ]]
+ touch arm64v8_User/src/jsk_robot/.git/CATKIN_IGNORE
+ for dir in $(find ${SOURCE_ROOT}/src/jsk_robot -maxdepth 1 -mindepth 1 -type d)
+ [[ ! arm64v8_User/src/jsk_robot/jsk_robot =~ jsk_unitree_robot|jsk_robot_common ]]
+ touch arm64v8_User/src/jsk_robot/jsk_robot/CATKIN_IGNORE
+ for dir in $(find ${SOURCE_ROOT}/src/jsk_robot -maxdepth 1 -mindepth 1 -type d)
+ [[ ! arm64v8_User/src/jsk_robot/jsk_denso_robot =~ jsk_unitree_robot|jsk_robot_common ]]
+ touch arm64v8_User/src/jsk_robot/jsk_denso_robot/CATKIN_IGNORE
+ for dir in $(find ${SOURCE_ROOT}/src/jsk_robot -maxdepth 1 -mindepth 1 -type d)
+ [[ ! arm64v8_User/src/jsk_robot/.github =~ jsk_unitree_robot|jsk_robot_common ]]
+ touch arm64v8_User/src/jsk_robot/.github/CATKIN_IGNORE
+ for dir in $(find ${SOURCE_ROOT}/src/jsk_robot -maxdepth 1 -mindepth 1 -type d)
+ [[ ! arm64v8_User/src/jsk_robot/jsk_baxter_robot =~ jsk_unitree_robot|jsk_robot_common ]]
+ touch arm64v8_User/src/jsk_robot/jsk_baxter_robot/CATKIN_IGNORE
+ for dir in $(find ${SOURCE_ROOT}/src/jsk_robot -maxdepth 1 -mindepth 1 -type d)
+ [[ ! arm64v8_User/src/jsk_robot/jsk_unitree_robot =~ jsk_unitree_robot|jsk_robot_common ]]
+ for dir in $(find ${SOURCE_ROOT}/src/jsk_robot -maxdepth 1 -mindepth 1 -type d)
+ [[ ! arm64v8_User/src/jsk_robot/jsk_aero_robot =~ jsk_unitree_robot|jsk_robot_common ]]
+ touch arm64v8_User/src/jsk_robot/jsk_aero_robot/CATKIN_IGNORE
+ for dir in $(find ${SOURCE_ROOT}/src/jsk_robot -maxdepth 1 -mindepth 1 -type d)
+ [[ ! arm64v8_User/src/jsk_robot/jsk_robot_common =~ jsk_unitree_robot|jsk_robot_common ]]
+ for dir in $(find ${SOURCE_ROOT}/src/jsk_robot -maxdepth 1 -mindepth 1 -type d)
+ [[ ! arm64v8_User/src/jsk_robot/jsk_fetch_robot =~ jsk_unitree_robot|jsk_robot_common ]]
+ touch arm64v8_User/src/jsk_robot/jsk_fetch_robot/CATKIN_IGNORE
+ for dir in $(find ${SOURCE_ROOT}/src/jsk_robot -maxdepth 1 -mindepth 1 -type d)
+ [[ ! arm64v8_User/src/jsk_robot/jsk_pr2_robot =~ jsk_unitree_robot|jsk_robot_common ]]
+ touch arm64v8_User/src/jsk_robot/jsk_pr2_robot/CATKIN_IGNORE
+ for dir in $(find ${SOURCE_ROOT}/src/jsk_robot -maxdepth 1 -mindepth 1 -type d)
+ [[ ! arm64v8_User/src/jsk_robot/jsk_kinova_robot =~ jsk_unitree_robot|jsk_robot_common ]]
+ touch arm64v8_User/src/jsk_robot/jsk_kinova_robot/CATKIN_IGNORE
+ '[' 0 -eq 0 ']'
+ tee arm64v8_build_user.log
++ id -u tsukamoto
+ docker run -it --rm -u 1000 -e INSTALL_ROOT=System -v /home/tsukamoto/ros/unitree_ws/src/jsk-ros-pkg/jsk_robot/jsk_unitree_robot/cross/arm64v8_System/ros1_dependencies:/opt/jsk/System/ros1_dependencies:ro -v /home/tsukamoto/ros/unitree_ws/src/jsk-ros-pkg/jsk_robot/jsk_unitree_robot/cross/arm64v8_System/Python:/opt/jsk/System/Python:ro -v /home/tsukamoto/ros/unitree_ws/src/jsk-ros-pkg/jsk_robot/jsk_unitree_robot/cross/arm64v8_System/ros1_inst:/opt/jsk/System/ros1_inst:ro -v /home/tsukamoto/ros/unitree_ws/src/jsk-ros-pkg/jsk_robot/jsk_unitree_robot/cross/arm64v8_System/ros1_dependencies_setup.bash:/opt/jsk/System/ros1_dependencies_setup.bash:ro -v /home/tsukamoto/ros/unitree_ws/src/jsk-ros-pkg/jsk_robot/jsk_unitree_robot/cross/arm64v8_System/system_setup.bash:/opt/jsk/System/system_setup.bash:ro -v /home/tsukamoto/ros/unitree_ws/src/jsk-ros-pkg/jsk_robot/jsk_unitree_robot/cross/arm64v8_User:/opt/jsk/User:rw -v /home/tsukamoto/ros/unitree_ws/src/jsk-ros-pkg/jsk_robot/jsk_unitree_robot/cross/rosinstall_generator_unreleased.py:/home/user/rosinstall_generator_unreleased.py:ro ros1-unitree:arm64v8 bash -c '    source /opt/jsk/System/system_setup.bash &&     env &&     set -xeuf -o pipefail &&     cd /opt/jsk/User &&     [ 0 -eq 0 ] || ROS_PACKAGE_PATH=src:${ROS_PACKAGE_PATH} /home/user/rosinstall_generator_unreleased.py jsk_unitree_startup unitreeeus --rosdistro melodic --exclude RPP --exclude mongodb_store | tee user.repos &&     [ 0 -eq 0 -o -z "$(cat user.repos)" ] || PYTHONPATH= vcs import src < user.repos &&     catkin build jsk_unitree_startup unitreeeus -s -vi         --cmake-args -DCATKIN_ENABLE_TESTING=FALSE     '
WARNING: The requested image's platform (linux/arm64/v8) does not match the detected host platform (linux/amd64) and no specific platform was requested
exec /bin/bash: no such file or directory
Makefile:18: recipe for target 'user' failed
make: *** [user] Error 1
```

--- a/jsk_unitree_robot/cross/build_user.sh
+++ b/jsk_unitree_robot/cross/build_user.sh
@@ -37,6 +37,10 @@ done
 # add unitree repos
 [ ${UPDATE_SOURCE_ROOT} -eq 0 ] || vcs import ${SOURCE_ROOT}/src < repos/unitree.repos
 
+# check if /proc/sys/fs/binfmt_misc/qemu-* is updated
+# See https://github.com/k-okada/jsk_robot/issues/61
+docker run -it --rm ros1-unitree:${TARGET_MACHINE} bash -c 'exit' ||  docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+
 # run on docker
 docker run -it --rm \
   -u $(id -u $USER) \

